Project2015To2017.Migrate2017.Tool/Program.cs

```diff
@@ -145,6 +149,12 @@ private static int ProcessArgs(ParseResult result)
 			OneOrMoreArguments()
 				.With("Target frameworks to be used instead of the ones in source projects", "frameworks"));
 
+		private static Option ForceTransformationsOption => Option(
+			"-f|--force-transformations",
+			"Force execution of transformations despite project conversion state by their specified names.",
```

Comment:
I'm not clear on what this message means, particularly `by their specified names`

Reply:
By transformations' specified names (like `"FileTransformation"`). Any ideas how to make this more clear?

Reply:
This means very precise knowledge of the internals of the tool. Do you see a real use case for this? I don't expect users of a command line tool to know which transformations are available unless we put that in the help file as well.

Reply:
I think we can merge this in, I still think it's a bad idea to make type names part of your CLI interface. But I'm guessing that very few people (if any) will use it so it might never result in issues being logged.